### PR TITLE
Don't rely on unstable flags

### PR DESCRIPTION
**Pull request by dpc**
Another attempt at #100

### CODE REVIEW

**Review by veehaitch — DISMISSED** [review]
Works like a charm!

File: build.nix [hunk 1]
```diff
@@ -239,6 +239,7 @@ let
       # threads as there are cores. This is often too much parallelism so we
       # reduce it to $NIX_BUILD_CORES if not specified by the caller.
       export RUST_TEST_THREADS="''${RUST_TEST_THREADS:-$NIX_BUILD_CORES}"
+      export CARGO_TARGET_DIR="$out"
```
Comment (nmattia) [inline build.nix:242]: I think you'll want `="out"` as opposed to `$out`! we want the directory `out`, not the build's output dir. Binaries are _then_ copied from `out` to `$out/bin`.

**Review by nmattia — COMMENTED** [review]
Thanks a lot! Just came back from holidays, sorry for the late review.

File: build.nix (hunk 1 above)
Comment (dpc) [reply build.nix]: Done.

**Comment by nmattia** [comment]
Alright let's try!

**Pull request merged**